packages/typespec-vscode/snippets.json

```diff
@@ -0,0 +1,37 @@
+{
+  "Alias": {
+    "prefix": "alias",
+    "body": "alias ${1:AliasName} = ${0:Expression};",
+    "description": "Alias definition"
+  },
+  "Enum": {
+    "prefix": "enum",
+    "body": "enum ${1:EnumName} {\n  ${0:Value},\n}",
```

Comment:
Value => value. Let's follow the naming convention in our snippet.

Reply:
I changed the names to align with typescript snippet: https://github.com/microsoft/vscode/blob/main/extensions/typescript-basics/snippets/typescript.code-snippets

Reply:
You can find typespec naming convension at https://typespec.io/docs/handbook/style-guide#naming-convention . Seems some naming in the snippet sill needs update according to that. thanks.

Reply:
> You can find typespec naming convension at [typespec.io/docs/handbook/style-guide#naming-convention](https://typespec.io/docs/handbook/style-guide#naming-convention) . Seems some naming in the snippet sill needs update according to that. thanks.

@RodgeFu Thanks. I changed the `name` placeholder to `Name`. I keep `type` as-is, since its first letter could be either uppercase (like `Pet`) or lowercase (like `string`).